### ISSUE TITLE
Customized navigation bar appearance is terrible in iOS 11.2

### ISSUE THREAD

**Issue by keithgee** [keithgee/Re-Resolver#35]
The customized navigation bars under iOS 11.2 look bad. See the top left of this screenshot.

![ugly_navigation_bar](https://user-images.githubusercontent.com/6785790/33515514-bf68ef5e-d732-11e7-80ad-dfd0183148fc.jpeg)

iOS 11.2 was released early this morning, unexpectedly, to fix a security and stability issue. I have a relatively quick fix for the navigation bar problem that involves removing the navigation bar customization and making it look more like a standard iOS app. This change would make the app look somewhat less like the original Resolver, but I think it makes sense to get this fix out quickly and then decide whether it is worth the ongoing maintenance costs to bring back the original look.

**Comment by keithgee** [comment]
This issue was automatically closed when I referenced it in a commit. It's fixed in the codebase but not deployed yet. 

I've submitted a build with the fix for iOS 11.2 to the iOS App Store.
 
Xcode 9.2, which is the first version that contains support for the iOS 11.2 version of the SDK, is still in beta. The App Store doesn't allow submissions built with beta versions of Xcode, so I built the submitted version with the current release of Xcode 9.1.

After testing, I believe that the built app works with all versions of iOS currently supported by Re-Resolver, 9.3 - 11.2, but I'm waiting to see if it is approved by the App Store reviewers.

**Comment by keithgee** [comment]
I canceled the app review this morning because of some issues when the iPhone app is run in the phone simulator on an iPad that supports iOS 11 but simulates the phone screen as the 3.5 inch iPhone 4S size. Under these circumstances there were problems with obscured content and controls on the select color and enter new choice screens.

I've made work arounds and will submit again for review once the new build is finished processing.
